Modules/ScormAicc/classes/class.ilObjSAHSLearningModuleGUI.php

```diff
@@ -768,15 +770,15 @@ function getTabs()
 		}
 
 		// export
-		if ($rbacsystem->checkAccess("edit_permission", "", $this->object->getRefId()))
+		if ($ilAccess->checkAccess("edit_permission", "", $this->object->getRefId()))
```

Comment:
Is the permission "edit_permission" correct here? 
Normally "write" permission is sufficient.